packages/block/test/eip4844block.spec.ts

```diff
@@ -14,10 +14,9 @@ import { BlockHeader } from '../src/header.js'
 import { fakeExponential, getNumBlobs } from '../src/helpers.js'
 import { Block } from '../src/index.js'
 
-import * as gethGenesis from './testdata/4844-hardfork.json'
+import gethGenesis from './testdata/4844-hardfork.json'
```

Comment:
TBH: on these cases I always totally have no idea what version to take and what difference this makes. 🤔

Reply:
For this one, if I understand how Typescript works with ESM, a namespace import like this converts the JSON into an object with a default export so it imported it as `{ "default": [ blocksData ] }` instead of `[ blocksData ]`.